src/averageAges.test.js

```diff
@@ -8,26 +8,30 @@ const {
 } = require('./averageAges');
 
 test('Average men age', () => {
-    expect(+calculateMenAverageAge(people).toFixed(2)).toBe(61.67);
+  expect(+calculateMenAverageAge(people).toFixed(2))
```

Comment:
1. why is + needed here?
2. how about we use?
toBeCloseTo
https://jestjs.io/docs/en/expect#tobeclosetonumber-numdigits